ui/eureka/eureka.js

```diff
@@ -1704,22 +1404,16 @@ let Options = {
             ja: 'フロストメーン',
             cn: '老虎',
           },
-          mobName: {
+          trackerName: {
```

Comment:
Is this intentional, vs keeping the trackerName the same?

Similar comments on other trackerName vs mobName changes below.

Reply:
I removed mobName because we don't care about the entities any more for tracking fates. The code relies only on the fate information.
I've updated all the trackerName entries with their current strings for all languages as of 15/04/2020.